app/parsers/ecs_sysmon.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from app.objects.secondclass.c_fact import Fact
+from app.objects.secondclass.c_relationship import Relationship
+from app.utility.base_parser import BaseParser
+
+
+class Parser(BaseParser):
+    """
+    This parser extracts information from sysmon events that
+     have been transformed via the Elastic Common Schema (ECS)
+    JSON documents (REF: https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html).
+    """
+
+    def parse(self, blob):
+        relationships = []
+        loaded = json.loads(blob)
+
+        # Do not parse facts if the result is an array (multiple results returned).
```

Comment:
this is kind of a dirty hack, but avoids making more sweeping changes in the rest of the codebase -- thus the long comment